### DISCUSSION TITLE
Return an empty LogPath from crictl inspect

### DISCUSSION THREAD

**Discussion by LinTechSo** [cri-o/cri-o#6602] [Q&A]
Hi all.

My problem is that, after i execute below command for getting container inspect information, the Log Path field in JSON output is None and i don't know why

would you please help me figure this out.
```
./crictl --runtime-endpoint /host/run/dockershim.sock inspect --output=json [container_ID] | jq
```

```
{
  "status": {
    "id": "70efc011fcb7d881eb036a865569eebe8bc24c182d6225279054f96f44",
      ........
    "mounts": [],
    "logPath": "",
    "resources": null
  }
}
```

**Comment by saschagrunert** [comment]
Hey @LinTechSo, it looks like you're running dockershim (aka docker), right? Are you sure this problem is related to CRI-O?
